source/change-streams/change-streams.rst

```diff
@@ -175,20 +175,34 @@ If an aggregate command with a ``$changeStream`` stage completes successfully, t
     updateDescription: Optional<UpdateDescription>;
 
     /**
-     * Always present for operations of type ‘insert’ and ‘replace’. Also
-     * present for operations of type ‘update’ if the user has specified ‘updateLookup’
-     * in the ‘fullDocument’ arguments to the ‘$changeStream’ stage.
+     * Always present for operations of type 'insert' and 'replace'. Also
+     * present for operations of type 'update' if the user has specified
+     * 'updateLookup' for the 'fullDocument' option when creating the change
+     * stream.
      *
-     * For operations of type ‘insert’ and ‘replace’, this key will contain the
-     * document being inserted, or the new version of the document that is replacing
-     * the existing document, respectively.
+     * For operations of type 'insert' and 'replace', this key will contain the
+     * document being inserted or the new version of the document that is
+     * replacing the existing document, respectively.
      *
-     * For operations of type ‘update’, this key will contain a copy of the full
+     * For operations of type 'update', this key will contain a copy of the full
      * version of the document from some point after the update occurred. If the
      * document was deleted since the updated happened, it will be null.
+     *
+     * Contains the point-in-time post-image of the modified document if the
+     * post-image is available and either "required" or "whenAvailable" was
```

Comment:
[opt-nit] This was already variable in the spec, but maybe we should pick `'` or `"`. `"required"` is `'required'` in `ChangeStreamOptions` below.

Reply:
Changed to single quotes.